dockerfiles/Dockerfile.android-common

```diff
@@ -1,25 +1,44 @@
-FROM openjdk:8-jdk-stretch
+FROM openjdk:11-jdk-buster
 
 RUN apt-get update
-RUN apt-get install -y gradle jq
+RUN apt-get install -y wget
+RUN apt-get install -y maven
+RUN apt-get install -y gnupg1
+RUN apt-get install -y cppcheck
+RUN apt-get install -y libncurses5
+RUN apt-get install -y jq
+RUN apt-get clean
```

Comment:
As I understand it, it's generally best to chain commands together in Docker files where reasonable to reduce the number of image layers and overall size.  E.g. something like:
RUN apt-get update && apt-get install -y wget maven gnupg1.... && apt-get clean